.github/comment.yml

```diff
@@ -3,11 +3,37 @@
     type: checkbox
     keywords:
       - "WaitForARMFeedback"
-    booleanFilterExpression: "!(ARMSignedOff||ARMChangesRequested||Approved-OkToMerge||WaitForARMRevisit||BreakingChangeReviewRequired)"
+    booleanFilterExpression: "!(ARMSignedOff||ARMChangesRequested||Approved-OkToMerge||WaitForARMRevisit||BreakingChangeReviewRequired||NewApiVersionRequired)"
     onCheckedLabels:
       - WaitForARMFeedback
       - ARMReview
-    onCheckedComments: "Hi, @${PRAuthor} your PR are labelled with <b> WaitForARMFeedback</b>. A notification email will be sent out shortly afterwards to notify ARM review board(armapireview@microsoft.com)."
+    onCheckedComments: >-
+      Hi @${PRAuthor}!<br/>
+      <br/>
+      The automation detected that in the `ARM API Review Checklist` you denoted that this PR requires an ARM review. As a result, the automation labelled your PR with `ARMReview` and `WaitForARMFeedback` labels. A notification email will be sent out shortly to notify ARM review board (armapireview@microsoft.com) to review your PR.
```

Comment:
FYI

>  A notification email will be sent out shortly to notify ARM review board (armapireview@microsoft.com) to review your PR.

This currently appears to be disabled. Details on the `RPC Governance - Azure SDK Bridge` Teams chat, 5/19/2023.

Reply:
@rkmanda I am thinking that for now we could replace:

> A notification email will be sent out shortly to notify ARM review board (armapireview@microsoft.com) to review your PR

With:

> The ARM team will review your PR in first-come first-served manner. To see your PR position in the PR review queue, see [this query](https://github.com/Azure/azure-rest-api-specs/pulls?q=is%3Aopen+is%3Apr+label%3AWaitForARMFeedback+-label%3AIDCDevDiv+draft%3Afalse). If you are not getting a review in timely manner, you can [mailto: ARM review board](mailto:armapireview@microsoft.com).

Sounds good?

Reply:
looks good!